Support actions in publishChecks Step (#85)
jenkinsci/checks-api-plugin: README.md

- publishChecks: you can publish checks directly in the pipeline script instead of depending on consumer plugins:

```
publishChecks name: 'example', title: 'Pipeline Check', summary: 'check through pipeline',
    text: 'you can publish checks in pipeline script',
    detailsURL: 'https://github.com/jenkinsci/checks-api-plugin#pipeline-usage',
    actions: [[label:'an-user-request-action', description:'actions allow users to request pre-defined behaviours', identifier:'an unique identifier']]
```

> *To use customized actions, you may need to define them on your own.

Reviewer: ```suggestion
*To use customized actions, you will need to write a Jenkins plugin
```

> For instance, if you want to add GitHub checks actions which are basically buttons on the checks report,

Reviewer: ```suggestion
If you want to add GitHub checks actions which are basically buttons on the checks report,
```

> you may need to extend [GHEventSubscriber](https://github.com/jenkinsci/github-plugin/blob/master/src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java) to handle the user requests,

Reviewer: ```suggestion
you need to extend [GHEventSubscriber](https://github.com/jenkinsci/github-plugin/blob/master/src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java) to handle the event,
```
